tox.ini

```diff
@@ -10,7 +10,7 @@ commands =
         pylint -j 0 -s n setup.py pytest_rts
         mypy setup.py pytest_rts --ignore-missing-imports
         safety check
-        pytest \
+        python3 -m pytest \
```

Comment:
what's the purpose of this change?

Reply:
After adding `__init__.py` to the `pytest_rts/tests/` folder I can't get tox working without changing `pytest` to `python3 -m pytest`. Causes some import error and I couldn't find any other solution

Reply:
It would be nice to understand the root cause before making random changes :)

Reply:
Import fails for finding `pytest_rts.tests` module in `tox` so would that module being missing from `PYTHONPATH` be the root cause? Another fix I found would be to the following to `tox.ini`:
```
setenv =
    PYTHONPATH = {toxinidir}
```

This also seems to collect coverage and adding `* /.tox/*` to `.coveragerc` to omit the duplicate files results in the following coverage report inside tox

``` 
----------- coverage: platform linux, python 3.8.5-final-0 -----------
Name                                       Stmts   Miss  Cover
--------------------------------------------------------------
pytest_rts/collect.py                          8      8     0%
pytest_rts/connection.py                      12      0   100%
pytest_rts/plugin.py                          68      7    90%
pytest_rts/pytest/collect_plugin.py           13     13     0%
pytest_rts/pytest/fake_item.py                 3      0   100%
pytest_rts/pytest/init_phase_plugin.py        12      0   100%
pytest_rts/pytest/mapper_plugin.py            29      3    90%
pytest_rts/pytest/normal_phase_plugin.py      11      6    45%
pytest_rts/pytest/update_phase_plugin.py      22      2    91%
pytest_rts/utils/common.py                    61      4    93%
pytest_rts/utils/git.py                       64      4    94%
pytest_rts/utils/mappinghelper.py            103      1    99%
pytest_rts/utils/selection.py                 38      0   100%
pytest_rts/utils/tables.py                     8      0   100%
pytest_rts/utils/testgetter.py                61      5    92%
--------------------------------------------------------------
TOTAL                                        513     53    90%
Coverage HTML written to dir htmlcov


=============================================================================================== 18 passed in 38.63s ===============================================================================================
_____________________________________________________________________________________________________ summary _____________________________________________________________________________________________________
  py38: commands succeeded
  congratulations :)
``` 

Does this look okay?